spec/support/crud.rb

```diff
@@ -22,11 +22,76 @@
 require 'support/crud/write'
 require 'support/crud/verifier'
 
+def crud_execute_operations(spec, test, num_ops, event_subscriber, expect_error,
+  client
+)
+  cache_key = "#{test.object_id}:#{num_ops}"
+  $crud_result_cache ||= {}
+  $crud_result_cache[cache_key] ||= begin
```

Comment:
Just to make sure I understand, the purpose of this cache is to avoid running the same test more than once and instead just reuse the results from the first run?

Reply:
That is correct.

Reply:
Awesome, thanks